sysroot/flags.bzl

```diff
@@ -36,17 +36,17 @@ def ldflags(arch, gcc_version):
         fail("unknown arch")
     return [
         "-B{toolchain_root}/bin",
-        "-B{sysroot}" + "/usr/{lib}".format(lib = lib),
-        "-B{sysroot}" + "/{arch_specific_prefix}{lib}".format(
+        "-B%sysroot%/usr/{lib}".format(lib = lib),
```

Comment:
interesting, any difference between `{}` and `%%` for python format strings that matters here?

Reply:
`%sysroot%` and `%workspace%` are part of the CC API. See https://github.com/bazelbuild/bazel/blob/a48e246e/src/main/java/com/google/devtools/build/lib/rules/cpp/CcToolchainProviderHelper.java#L234-L254.

I added this link to the documentation in this PR since the published documentation from Bazel doesn't expand on them, though this comment on their code is pretty good.